plugins/generic/usageEvent/UsageEventPlugin.inc.php

```diff
@@ -286,6 +286,18 @@ function _buildUsageEvent($hookName, $args) {
 			$request, null, $canonicalUrlPage, $canonicalUrlOp, $canonicalUrlParams
 		);
 
+		// Make sure we log the server name and not aliases.
+		$configBaseUrl = Config::getVar('general', 'base_url');
+		$requestBaseUrl = $request->getBaseUrl();
+		if ($requestBaseUrl !== $configBaseUrl) {
+			// Make sure it's not an url override (no alias on that case).
+			if (!in_array($requestBaseUrl, Config::getContextBaseUrls()) && 
+					$requestBaseUrl !== Config::getVar('general', 'base_url[index]')) {
```

Comment:
And will the config's general base_url work for stat's processing at that point, even if mod_rewrite obscured the journal's shortname in the URI?  If so, then we are fine.  If not, I think the code should look more like:
`$canonicalUrl = str_replace($requestBaseUrl, Config::getVar('general', 'base_url['.$journalKey.']') ? Config::getVar('general', 'base_url['.$journalKey.']') : $configBaseUrl, $canonicalUrl);`
(untested)